app/src/full/AndroidManifest.xml

```diff
@@ -12,9 +12,15 @@
 		<meta-data
 			android:name="com.google.firebase.messaging.default_notification_color"
 			android:resource="@color/color_base" />
-
-		<service android:name=".common.FirebaseMessagingService">
-			<intent-filter>
+		<meta-data
+			android:name="com.google.firebase.messaging.default_notification_channel_id"
+			android:value="@string/General_AppName" />
+		<meta-data
+			android:name="firebase_analytics_collection_deactivated"
+			android:value="true" />
+		<service android:name=".common.FirebaseMessagingService"
+			android:exported="false">
```

Comment:
android:exported Whether or not the broadcast receiver can receive messages from sources outside its application — "true" if it can, and "false" if not. If "false", the only messages the broadcast receiver can receive are those sent by components of the same application or applications with the same user ID.